### PR TITLE
Countydata

### PR DESCRIPTION
**Pull request by sjsrey**
This refactors the county dataset to be in line with the other book example datasets. The final geopackage is now included in the repo, and the instructions in the readme document the creation of that file.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,16 @@ lab:
                -p 8888:8888 \
                -v ${PWD}:/home/jovyan/work \
                darribas/gds_dev:5.0
+lablinuxsr:
+	docker run --rm \
+	--user root \
+	-e NB_UID=1001 \
```
Comment (ljwolf) [inline Makefile:11]: Could we adopt this in the standard lab? Does this break the lab for others on windows?

**Comment by ljwolf** [comment]
This hits many files just from the jupytext version bump, it seems...

**Comment by sjsrey** [comment]
> This hits many files just from the jupytext version bump, it seems...

Yah, I think most are in docs/

**Comment by sjsrey** [comment]
replaced by #119

**Pull request closed without merging**